roles/ceph-osd/tasks/system_tuning.yml

```diff
@@ -32,6 +32,32 @@
     when: ursula_os == 'rhel'
   when: ceph.disable_transparent_hugepage
 
+- name: create ursula tuned profile dir
+  file: path=/etc/tuned/ursula state=directory
+  when: ursula_os == 'rhel'
+
+# tune disk read-ahead for ceph
+# if there are more tunings in future,
+# we should move the common part(main section) into common role
+- name: set disk read-ahead for ceph
+  ini_file:
```

Comment:
/etc/tuned/ursula/tuned.conf could be written by other team too, such as by nework team for network performance tuning. Each team can make their tuning existing in tuned.conf, but they can't remove other teams' tuning. Template is not able to implement this, so I use ini_file module.
You may want to say each team can create their separated tuning file, and use `include` to merge these tunings.
Unfortunately, I read tuned documents, tuned doesn't support more than one `include`.